src/scss/components/_donors.scss

```diff
@@ -1,29 +1,41 @@
-.donors_info {
-  @include grid;
-}
-
 .donors {
   position: relative;
   overflow: hidden;
 }
 
+.donors_info {
+  @include grid;
+
+  display: block;
+}
+
 .donors_image {
   width: 100%;
 }
 
 .donors_description {
-  @include heading-3;
+  @include heading-4;
 
+  font-size: 3rem;
   font-weight: var(--font-med);
-  @include col(2, 7);
+
+  @include breakpoint(md) {
+    @include col(2, 7);
+  }
 }
 
 .donors_link_container {
   @include link-typography;
-  @include col(2, 4);
 
   display: flex;
+  justify-content: space-around;
   padding: 2.5rem 0 19.5rem;
```

Comment:
Move this padding to the `.donors` class, and only use `padding-bottom`. Like this
```
padding-bottom: 19.5rem;
```